interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git stands for global information technology, which was created a to store data. Austin also has a unique name for it. I am assuming the difference between git and Github is that the hub has a UI attached to it.
 
-  Researched answer:
+  Researched answer: Git is used primarily from terminal or a command line, while Github is provides a website that allows interaction through an interface instead of just a terminal. There are additional tools added into Github that improves the ease of access to a repository.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: True and False evaluations, which is also 1 or 0.
 
-  Researched answer:
+  Researched answer: Any evaluation that has the potential for a yes or no output
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: An index is a number, while a value can hold different qualities.
 
-  Researched answer:
+  Researched answer: An index is the position in a string related to a numerical value. Values can relate to strings, numbers, and floats.
 
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer: Built-in methods are previously coded commands that grant ease of use for programmers to simplify their algorithms.
 
-  Researched answer:
+  Researched answer: Built-in methods can be strings, math, dates, and arrays that have been created by previous developers to expedite the coding process for others. Without these we would have to code out every single command from scratch.
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: The driver is responsible for typing out the code, while the navigator ensures proper execution and research of the code. Basically the driver is listening to the guidelines of the navigator in order to reduce the mental workload on each individual.
 
-  Researched answer:
+  Researched answer: It is mores simply suppose to be the driver types as the navigator reviews the code as they progress.
```

Comment:
Navigator is in charge of: quality, research, plan, and communication (though that one is a bit on both)